lib/model.d.ts

```diff
@@ -107,17 +109,17 @@ export interface ScopeOptions {
 export type WhereOptions = WhereAttributeHash | AndOperator | OrOperator | Where
 
 /**
- * Example: `$any: [2,3]` becomes `ANY ARRAY[2, 3]::INTEGER`
+ * Example: `[Op.any]: [2,3]` becomes `ANY ARRAY[2, 3]::INTEGER`
  *
  * _PG only_
  */
 export interface AnyOperator {
-    $any: (string | number)[]
+    [Op.any]: (string | number)[]
```

Comment:
This is a breaking change and should probably not be merged till v5 is out

Reply:
Correct, this PR is meant to be for v5.